File: Java/LeapYear.java
public class LeapYear{
    static boolean checkLeapYear(int year){
        if((year %4==0 && year % 100 != 0 ) || (year%400 ==0)){
            return true;
        }
        return false;
    }
    public static void main(String[] args){
        int[] years={2023,2024,1900,2000};
        for (int year : years) {
            if (checkLeapYear(year)) {
                System.out.println(year + ": Leap Year");
            } else {
                System.out.println(year + ": Not a Leap Year");
            }
        }
    }
}

File: Python/LeapYear.py
def checkLeapYear(year):
    if(year %4==0 and year%100 !=0) or(year %400 ==0):
        return True
    return False

y=[2023,2024,1900,2000]
for i in y:
    print(f"{i} : {'Leap Year' if checkLeapYear(i) else 'Not a Leap Year'}")